upstream-sha256: 7da5885241642504e48eefb7b4d6625550ef3527e0a57db738aa4515b943fb0a
actual-sha256: cad7dd6577b948f93f78aff06ce94d7e559479bc7687a5f45c28b4dc6f65fb58
--- upstream.expected
+++ tolk.actual
@@ -20,5 +20,5 @@
   float val9 = (*(data1_1000+(alu0+900)));
   int alu1 = (lidx1+(gidx0<<1)+(gidx1*20)+(lidx0*10));
   float val10 = (*(data2_100+alu1));
-  *(data0_100+alu1) = (val0+val1+val2+val3+val4+val5+val6+val7+val8+val9+val10);
+  *(data0_100+alu1) = (0.0f+val0+val1+val2+val3+val4+val5+val6+val7+val8+val9+val10);
 }
